### PR TITLE
Update style imports and demo file names

### LINKED ISSUE #8: Incorrect css file path being used in examples
### Please verify the version of auro-dialog you have installed

1.0.2

### Please describe the bug

1. See `<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/auro-dialog@latest/dist/style-unformatted.css" />` being used in `editCloseButton.html`
2. See how `style-unformatted.css` is not included in `dist` directory

### Reproducing the error

This issue cannot be reproduced on the Auro docsite

### Expected behavior

We do not use this script in the example file

### Exit criteria

Script is removed from example file

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #8 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Update references to `unformatted` style links
* Update demo page references

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/publish_component.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ labels: 'auro-dialog, not-reviewed'
     - [ ] All style code follows [Auro's Style Guide](http://auro.alaskaair.com/webcorestylesheets/guidelines)
     - [ ] Automated tests created
 1. __Documentation__
-    - `demo.md`
+    - `index.md`
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/publish_component.md]: This whole document can be deleted. The element is already published.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
We need to talk about this update. This feature _was_ supported and I am not sure why we are deleting code?

**Comment by jordanjones243** [comment]
> We need to talk about this update. This feature _was_ supported and I am not sure why we are deleting code?

The `unformatted` feature is still supported and the examples still work as expected. There was no need for the link to the stylesheet in the example files anymore because we already had `style-unformatted.css` being imported into the component's JS file.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**